themes-default/slim/views/partials/showheader.mako

```diff
@@ -10,64 +10,47 @@
-                <%include file="/partials/seasonEpisode.mako"/>
+                Manual search for:<br>
+                <app-link href="home/displayShow?indexername=${show.indexer_name}&seriesid=${show.series_id}" class="snatchTitle">{{ show.title }}</app-link> / Season {{ season }}<template v-if="episode"> Episode {{ episode }}</template>
             </span>
         </div>
-    % else:
-        % if season_results:
-            ##There is a special/season_0?##
-            % if int(season_results[-1]["season"]) == 0:
-                <% season_special = 1 %>
-            % else:
-                <% season_special = 0 %>
-            % endif
-            % if not app.DISPLAY_SHOW_SPECIALS and season_special:
-                <% lastSeason = season_results.pop(-1) %>
-            % endif
+        <template v-if="$route.name !== 'snatchSelection' && show.seasons && show.seasons.length >= 1">
             <div id="show-specials-and-seasons" class="pull-right">
-                <span class="h2footer display-specials">
-                % if season_special:
-                    Display Specials: <a class="inner" style="cursor: pointer;">${'Hide' if app.DISPLAY_SHOW_SPECIALS else 'Show'}</a>
-                % endif
+                <span class="h2footer display-specials" v-if="show.seasons.find(({ season }) => season === 0)">
+                    Display Specials: <a @click="toggleSpecials()" class="inner" style="cursor: pointer;">{{ config.layout.show.specials ? 'Hide' : 'Show' }}</a>
                 </span>
 
                 <div class="h2footer display-seasons clear">
                     <span>
-                    % if (len(season_results) > 14):
-                        <select id="seasonJump" class="form-control input-sm" style="position: relative">
-                            <option value="jump">Jump to Season</option>
-                        % for seasonNum in season_results:
-                            <option value="#season-${seasonNum["season"]}" data-season="${seasonNum["season"]}">${'Season ' + str(seasonNum["season"]) if int(seasonNum["season"]) > 0 else 'Specials'}</option>
-                        % endfor
-                        </select>
-                    % else:
-                        Season:
-                        % for seasonNum in season_results:
-                            % if int(seasonNum["season"]) == 0:
-                                <app-link href="#season-${seasonNum["season"]}">Specials</app-link>
-                            % else:
-                                <app-link href="#season-${seasonNum["season"]}">${str(seasonNum["season"])}</app-link>
-                            % endif
-                            % if seasonNum != season_results[-1]:
-                                <span class="separator">|</span>
-                            % endif
-                        % endfor
-                    % endif
+                        <template v-if="show.seasons && show.seasons.length >= 13">
```

Comment:
You already checked for `show.seasons` above, so you know it's not `undefined` / `null`.
Plus, same note as above, no need for the `<template>` wrapper, put it on the `<select>` element,
**or better yet**, the `<span>` element it's currently in.
As long as you don't use `<element v-for="..." v-if="...">` you should be good to go.

Reply:
I could be misreading that, but to me it looks like you changed the logic here:
```py
len(season_results) > 14
```
is not the same as
```js
show.seasons.length >= 13
```

```
14 > 14 : false
14 >= 13 : true
```

Reply:
So it should be `>= 15`. Will change.